ui/apps/platform/src/Containers/DeferralConfiguration/VulnerabilitiesConfiguration.tsx

```diff
@@ -23,8 +26,9 @@ function VulnerabilitiesConfiguration() {
                 </Split>
             </div>
             <Divider component="div" />
-            <PageSection variant="light">
+            <PageSection variant="light" component="div">
                 <Title headingLevel="h2">Configure deferral times</Title>
+                <VulnerabilitiesConfigForm />
```

Comment:
I have a thought on this. Instead of having a separate `VulnerabilitiesConfigForm` component, what if we displayed its contents directly here? My concern with the separate component is that it makes navigation a bit jumpy. Having everything in one place might make things clearer and quicker to grasp. What do you think?

Reply:
Also, how about setting up a new directory under `/Containers/DeferralConfiguration/`? We could name it `forms` or` components`, and then shift both NumericSetting and BooleanSetting components into it.

Reply:
RE: comment one - Sure, the reason for separating the form from the container initially was so that I could conditionally not render the form until the `config` loaded from the API, then pass that to Formik as the `initialValues`. I found a better solution that doesn't require that so I agree it would be clearer to move the entire form inline to simplify this component.

I'm on the fence about moving the NumericSetting and BooleanSetting components out though, those currently are specific to this form, mostly to get the correct layout. I'm not sure if we would reuse them in other type of deferral config but if so it would make sense to refactor as a follow up.